projects/plugins/boost/app/modules/cache/pre-wordpress/Boost_Cache.php

```diff
@@ -193,13 +193,25 @@ public function delete_on_comment_edit( $comment_id, $commentdata ) {
 	 */
 	public function delete_on_comment_post( $comment_id, $comment_approved, $commentdata ) {
 		$post = get_post( $commentdata['comment_post_ID'] );
-
+		error_Log( "delete_on_comment_post: $comment_id, $comment_approved, {$post->ID}" ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
```

Comment:
I know `error_log`'s can be useful, but perhaps we need to create an utility helper so that we don't need to `error_log` everywhere. This is probably an improvement for another PR.

Could do something like `boost_cache_log("delete_comment_on_post", $comment_id, $comment_approved, $post_id)`. That would also allow us to eventually change where the log is shipped to.

Reply:
There is a Logger class now. I need to start using that.